sia-react-native/Backend.js

```diff
@@ -55,9 +55,8 @@ class Backend extends React.Component {
         events.push(new event(doc.id, doc.data().name, doc.data().description,
           doc.data().location, doc.data().start_date_time, doc.data().end_date_time, []))
       });
-
-
     });
+    return events
```

Comment:
Is it possible to return events and eventList separately? I haven't been able to find any documentation, but if this works LGTM

Reply:
Had that there for debugging. I'll Address it